go.mod

```diff
@@ -5,7 +5,7 @@ go 1.13
 require (
 	github.com/cloudevents/sdk-go v1.1.2
 	github.com/dghubble/sling v1.3.0
-	github.com/edgexfoundry/app-functions-sdk-go v1.0.1-dev.43
-	github.com/edgexfoundry/go-mod-core-contracts v0.1.57
-	github.com/stretchr/testify v1.5.1
+	github.com/edgexfoundry/app-functions-sdk-go v1.2.1-dev.9
+	github.com/edgexfoundry/go-mod-core-contracts v0.1.60
+	github.com/stretchr/testify v1.6.1
```

Comment:
@lenny-intel , good catch, I have updated the dependencies. I was surprised that this repo is still on app-functions-sdk-go v1.0.1-dev.43 still though. Any concerns about the upgrade?

Reply:
No concerns about bringing it up to latest. @rsdmike , this repo is a great candidate for DependABot.